scripts/lib/CIME/SystemTests/err.py

```diff
@@ -3,39 +3,33 @@
 ERR tests short term archiving and restart capabilities
 """
 from CIME.XML.standard_module_setup import *
-from CIME.SystemTests.ers import ERS
-
-import shutil, glob
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
+from CIME.SystemTests.irt import IRT
+from CIME.case_st_archive import restore_from_archive
+from CIME.utils import ls_sorted_by_mtime
 
 logger = logging.getLogger(__name__)
 
-class ERR(ERS):
+class ERR(IRT):
```

Comment:
I find it more confusing than helpful that this inherits from IRT, particularly given that you override the most important method of IRT (`_case_one_custom_postrun_action`): this gives a false impression that this does something similar to IRT, when really the only functionality that it reuses seems to be the basic restart test stuff in _case_one_setup and _case_two_setup. I'd rather just see those duplicated. Or, if you'd like to avoid that duplication, you could make a base class for restart tests that has the body of IRT's _case_two_setup, and then have both IRT and ERR inherit from that restart test base class.